Makefile

```diff
@@ -78,14 +78,14 @@ stop-kubernetes-master:
 	-docker rm -f calico-kubelet-master
 
 	# Remove any pods that the old kubelet may have started.
-	-docker rm -f $$(docker ps | grep k8s_ | awk '{print $$1}')
+	-docker ps -af name=k8s_ | awk 'NR < 2 {next}{print $$1}' | xargs -r docker rm -f
```

Comment:
Hm, I think the use of `xargs -r` breaks this on BSD / Mac.  See here: http://stackoverflow.com/questions/8803987/what-is-the-equivalent-to-xargs-r-under-osx

I think this needs to be tweaked so the tests can successfully run on a Mac.